README.md

```diff
@@ -112,7 +112,35 @@ set :bucket_write_options, {
 }
 ```
 
-See aws-sdk [S3Client.put_object doc](http://docs.aws.amazon.com/AWSRubySDK/latest/AWS/S3/Client.html#put_object-instance_method) for all available options.
+You can also set write options for files matching specific [patterns](https://ruby-doc.org/core-2.3.0/Dir.html#method-c-glob) using:
+
+```ruby
+set :object_write_options, {
+  'index.html' => { cache_control: 'no-cache' }
+}
+```
+
+A more advanced scenario
+
+```ruby
+set :object_write_options, {
+  'assets/**' => { cache_control: 'public, max-age=86400' },
+  'index.html' => { cache_control: 'no-cache' }
+}
+```
+
+Please note that pattern matching for `object_write_options` is evaluated in the order of definition and overrides on match down the chain. For example defining
```

Comment:
Is `bucket_write_options` also overridden by `object_write_options`?

Reply:
Yes, `object_write_options` are evaluated *after* `bucket_write_option`. Basically `**` pattern at the first place equals to the `bucket_write_options` functionality, but I didn't want to break the API.

Reply:
Nonetheless, I'll update the docs, thanks for pointing out!

Reply:
LGTM